src/sbt-test/sbt-tpolecat/scalacOptions/build.sbt

```diff
@@ -1,5 +1,6 @@
-import scala.util.Try
+import _root_.io.github.davidgregory084.JavaMajorVersion
```

Comment:
IMO along with #65 this is another point in favour of changing the package names in this project